index.html

```diff
@@ -3720,6 +3720,18 @@ <h1>
 Resolution
     </h1>
 
+    <p class="issue atrisk" data-number="549">
+The Working Group is unsure if there will be enough implementation experience
+for the DID Resolution section. We are seeking feedback from the implementation
+community as to whether they prefer to do this work now, or if they would
+prefer that this section is published as a NOTE and taken up in a future
+W3C DID Resolution Working Group. If there is support for publishing the
+DID Resolution section as a NOTE during the W3C Candidate Recommendation
+process, this section will be migrated to a new document and published as a
+NOTE before the DID Core specification proceeds to the W3C Proposed
+Recommendation stage.
```

Comment:
```suggestion
The Working Group is unsure if there will be enough implementation experience
for the DID Resolution section. We are seeking feedback from the implementation
community as to whether they prefer to do all of this work now, or if they would
prefer that this section is, or parts of the section are, rewritten to be non-normative, 
or published as a NOTE and taken up in a future W3C DID Resolution Working 
Group. If there is support for rewriting a subset of the DID Resolution section,
or publishing any part of it as a NOTE during the W3C Candidate Recommendation 
process, this section will be modified and/or published as a NOTE appropriately 
before the DID Core specification proceeds to the W3C Proposed 
Recommendation stage.
```
Add language requested to allow multiple options to be taken with this section.